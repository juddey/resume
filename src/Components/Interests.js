import React from 'react'
import Typography from '@material-ui/core/Typography'
// import styles from './Styles/InterestsStyle'
import { Images } from '../Themes'

const data = [
  { image: Images.beer, label: 'Beer' },
  { image: Images.church, label: 'Church' },
  { image: Images.drink, label: 'Coffee' },
  { image: Images.laptop, label: 'Coding' },
  { image: Images.shoes, label: 'Running' }
]

const Interests = () => (
  <div style={{ display: 'flex', flexWrap: 'wrap' }}>
    {data.map((x, i) => (
      <div
        key={i}
        style={{
          width: 75,
          height: 75,
          display: 'flex',
          alignItems: 'center',
          flexDirection: 'column'
        }}
      >
        <img src={x.image} style={{ width: 50, height: 50 }} />
        <Typography>{x.label}</Typography>
      </div>
    ))}
  </div>
)

export default Interests
