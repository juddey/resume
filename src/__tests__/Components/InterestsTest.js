import React from 'react'
import {
  createComponentWithIntl,
  shallowWithIntl
} from '../../../src/Support/Utils'
import Interests from '../../Components/Interests'

const div = document.createElement('div')
const wrapper = shallowWithIntl(<Interests />)

test('component exists', () => {
  expect(wrapper.length).toBe(1)
})

test('component structure', () => {
  expect(wrapper.name()).toEqual('div')
  expect(wrapper.children().length).toEqual(5)
})

test('renders to snapshot', () => {
  const tree = createComponentWithIntl(<Interests />, div).toJSON()
  expect(tree).toMatchSnapshot()
})
