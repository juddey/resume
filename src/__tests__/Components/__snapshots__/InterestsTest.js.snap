// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to snapshot 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexWrap": "wrap",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "height": 75,
        "width": 75,
      }
    }
  >
    <img
      src="beer.png"
      style={
        Object {
          "height": 50,
          "width": 50,
        }
      }
    />
    <p
      className="MuiTypography-root-1 MuiTypography-body1-10"
    >
      Beer
    </p>
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "height": 75,
        "width": 75,
      }
    }
  >
    <img
      src="church.png"
      style={
        Object {
          "height": 50,
          "width": 50,
        }
      }
    />
    <p
      className="MuiTypography-root-1 MuiTypography-body1-10"
    >
      Church
    </p>
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "height": 75,
        "width": 75,
      }
    }
  >
    <img
      src="hot-drink.png"
      style={
        Object {
          "height": 50,
          "width": 50,
        }
      }
    />
    <p
      className="MuiTypography-root-1 MuiTypography-body1-10"
    >
      Coffee
    </p>
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "height": 75,
        "width": 75,
      }
    }
  >
    <img
      src="laptop.png"
      style={
        Object {
          "height": 50,
          "width": 50,
        }
      }
    />
    <p
      className="MuiTypography-root-1 MuiTypography-body1-10"
    >
      Coding
    </p>
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "height": 75,
        "width": 75,
      }
    }
  >
    <img
      src="shoes.png"
      style={
        Object {
          "height": 50,
          "width": 50,
        }
      }
    />
    <p
      className="MuiTypography-root-1 MuiTypography-body1-10"
    >
      Running
    </p>
  </div>
</div>
`;
